apps/browser/src/tools/popup/send-v2/send-created/send-created.component.html

```diff
@@ -1,6 +1,11 @@
 <main class="tw-top-0">
   <popup-page>
-    <popup-header slot="header" [pageTitle]="'createdSend' | i18n" showBackButton>
+    <popup-header
+      slot="header"
+      [pageTitle]="'createdSend' | i18n"
+      showBackButton
+      [backAction]="close.bind(this)"
```

Comment:
⛏️ This can just be close
```suggestion
      [backAction]="close"
```

Reply:
@djsmith85 Binding to the current context is required here, otherwise the popup header component will attempt to call `this.close` which doesn't exist in `PopupHeaderComponent`. There's a similar implementation [here](https://github.com/bitwarden/clients/blob/4b6c7bf0df1012aea1c820d6de28898fc64c17b2/apps/browser/src/vault/popup/components/vault-v2/add-edit/add-edit-v2.component.html#L5).